### DISCUSSION TITLE
how to add a image field in the [advanced block] [basic line]?

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#16901] [How to]
how to add a image field in the [advanced block] [basic line]?

**Comment by mariuszkrzaczkowski** [comment]
there is no such field type for advanced block

**Reply by newminder** [reply]
any way to show the product image?

**Reply by mariuszkrzaczkowski** [reply]
in PDF you can show the picture
https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/app/TextParser/ProductsTableImages.php

**Reply by newminder** [reply]
what if I want to edit the editviewvalue.tpl and limit it with 
{if $field-get('columnName') =="image"}  <img scr="{}" style="width:30px;height;auto;">
how should I do? and how to fill the scr...
